docs/migration-guide.md

```diff
@@ -0,0 +1,112 @@
+---
+id: migration
+title: Migration guide
+---
+This tutorial guide your throught to one way to migrate your earlier version of express-validator to version 6.
```

Comment:
You mean "through", don't you?

Reply:
Thanks for detailed feedback. Just updated the documentation.